app/src/main/java/ithaca_transit/android/cornellappdev/com/ithaca_transit/Models/Place.java

```diff
@@ -52,14 +54,18 @@ public String getName() {
 
     public String getPlaceID() {
         return placeID;
```

Comment:
in Android, this should be `Id` and not `ID`

Reply:
Changed the variable name, thanks!